src/topology/metric_space/isometric_smul.lean

```diff
@@ -41,6 +46,11 @@ instance has_isometric_smul.to_has_continuous_const_smul [pseudo_emetric_space X
   [has_isometric_smul M X] : has_continuous_const_smul M X :=
 ⟨λ c, (isometry_smul X c).continuous⟩
 
+@[priority 100, to_additive]
+instance has_isometric_smul.opposite_of_comm [comm_semigroup M] [pseudo_emetric_space M]
+  [has_isometric_smul M M] : has_isometric_smul Mᵐᵒᵖ M :=
+⟨λ c x y, by simpa only [mul_comm c.unop, smul_eq_mul] using (isometry_smul M c.unop x y)⟩
```

Comment:
This should generalize to the `is_central_scalar` case, since `comm_semigroup M` implies `is_central_scalar M M M`